pettingzoo/mpe/simple/simple.py

```diff
@@ -51,8 +52,9 @@
 from .._mpe_utils.simple_env import SimpleEnv, make_env
 
 
-class raw_env(SimpleEnv):
+class raw_env(SimpleEnv, EzPickle):
     def __init__(self, max_cycles=25, continuous_actions=False, render_mode=None):
+        EzPickle.__init__(self, max_cycles, continuous_actions, render_mode)
```

Comment:
Use keyword arguments not positional arugments as it is easier when debugging to understand the value, `max_cycles=max_cycles`

Reply:
Thanks for the review, will incorporate these changes into shimmy too as I think there were some cases where positional args were used